docs/modules/ROOT/pages/metrics.adoc

```diff
@@ -0,0 +1,1678 @@
+
+|`tcp.connection.in/out.opsInterested`
+|count
+|
+
+|`tcp.connection.in/out.opsReady`
+|count
+|
+
+|`tcp.connection.in/out.ownerId`
+|count
+|Owner ID of this `NioPipeline`, -1 if the pipeline is being migrated (owner is null)
+
+|`tcp.connection.in/out.processCount`
+|count
+|Number of time the `NioPipeline.process()` method has been called
+
+|`tcp.connection.in/out.startedMigrations`
+|count
+|Number of started migrations on this pipeline
+
+|`tcp.connection.in.bytesRead`
+|bytes
+|Total size of frames read on this inbound pipeline
+
+|`tcp.connection.in.idleTimeMs`
+|ms
+|Idle time that indicates how long since the last read on this inbound nio pipeline
+
+|`tcp.connection.in.normalFramesRead`
+|count
+|Number of priority frames read on this inbound nio pipeline
+
+|`tcp.connection.in.priorityFramesRead`
+|count
+|Number of priority frames read
+
+|`tcp.connection.out.bytesWritten`
+|bytes
+|Total amount of written frames on this outbound pipeline
+
+|`tcp.connection.out.idleTimeMillis`
+|ms
+|Idle time that indicates how long since the last write on this outbound nio pipeline
+
+|`tcp.connection.out.normalFramesWritten`
+|count
+|Number of written normal frames on this outbound nio pipeline
+
+|`tcp.connection.out.priorityFramesWritten`
+|count
+|Number of priority frames written into this nio pipeline
+
+|`tcp.connection.out.priorityWriteQueuePendingBytes`
+|bytes
+|Total size of priority frames pending in the write queue
+
+|`tcp.connection.out.priorityWriteQueueSize`
+|count
+|Number of priority frames pending in the write queue
+
+|`tcp.connection.out.scheduled`
+|count
+|Ordinal of enum state of this outbound pipeline: 0 -> UNSCHEDULED, 1 -> SCHEDULED, 2 -> BLOCKED, 3 -> RESCHEDULE
+
+|`tcp.connection.out.writeQueuePendingBytes`
+|bytes
+|Total size of normal frames pending in the write queue
+
+|`tcp.connection.out.writeQueueSize`
+|count
+|Number of normal frames pending in the write queue
+
+|`tcp.inputThread/outputThread.bytesTransceived`
+|bytes
+|Amount of transceived data on this NioThread
+
+|`tcp.inputThread/outputThread.completedTaskCount`
+|count
+|Total number of completed tasks on this NioThread
+
+|`tcp.inputThread/outputThread.eventCount`
+|count
+|Total number of the connections accepted by TcpServerAcceptor
+
+|`tcp.inputThread/outputThread.framesTransceived`
+|count
+|Number of transceived frames on this NioThread
+
+|`tcp.inputThread/outputThread.idleTimeMillis`
+|ms
+|Idle time that indicates how long since the last read/write
+
+|`tcp.inputThread/outputThread.ioThreadId`
+|count
+|Thread ID of this NioThread
+
+|`tcp.inputThread/outputThread.priorityFramesTransceived`
+|count
+|Number of transceived priority frames
+
+|`tcp.inputThread/outputThread.processCount`
+|count
+|Number of processed `NioPipeline`s on this NioThread
+
+|`tcp.inputThread/outputThread.selectorIOExceptionCount`
+|count
+|Number of times that io exceptions are thrown during selection
```

Comment:
... that I/O exceptions..

(just being consistent with the other I/O mentions in the documentation)